packages/components/src/molecules/Panels/ResizableColumnsPanel.tsx

```diff
@@ -39,17 +45,57 @@ const ResizableColumnsPanel: React.FC<ResizableColumnsPanelType> = (props) => {
           onStopResize={onStopResizeCenter}
           flex={layout?.center}
         >
-          <S.Pane>{center}</S.Pane>
+          <StyledPane>{center}</StyledPane>
         </ReflexElement>
       )}
 
       {center && <ReflexSplitter propagate={Boolean(left)} />}
 
       <ReflexElement minSize={minPaneWidth} onStopResize={onStopResizeRight}>
-        <S.Pane>{right}</S.Pane>
+        <StyledPane>{right}</StyledPane>
       </ReflexElement>
     </ReflexContainer>
   );
 };
 
 export default ResizableColumnsPanel;
+
+// Styled Components
+
+const StyledPane = styled.div`
+  position: relative;
+  height: 100%;
+  width: 100%;
+  overflow-y: auto;
+  overflow-x: hidden;
+`;
+
+const StyledLeftPane = styled(StyledPane)<{ $leftClosable: boolean }>`
+  background-color: ${Colors.grey10};
+
+  ${({ $leftClosable }) => {
+    if ($leftClosable) {
+      return `position: static;`;
+    }
+  }}
+`;
+
+const StyledLeftReflexElement = styled(ReflexElement)<{ $leftClosable: boolean }>`
+  ${({ $leftClosable }) => {
+    if ($leftClosable) {
+      return `overflow: visible !important;`;
+    }
+  }}
+`;
+
+// Utils
+
+const makeOnStopResize = (position: "left" | "center" | "right", onStopResize?: OnStopResize) => {
+  return (args: HandlerProps) => {
+    const flex = args.component.props.flex;
+
+    if (flex && onStopResize) {
```

Comment:
if this check fails this function does not do anything so in this case might be better to check these before function call and not call this function if they are not available and remove checks from here and remove optional syntax on parameters.

Reply:
I don't think there will be a real case where the `onStopResize` won't exist. We could just return if the `flex` and `onStopResize` doesn't exist

Reply:
if there is not a real case lets remove that check?

Reply:
The typescript will give an error if we don't check as `onStopResize` prop is optional

Reply:
if you check onStopResize before calling function typescript will not give warning also we need to remove onStopResize?: OnStopResize question mark there since it is not optional. Btw please do not get me as nit picker just better to remove but feel free to merge like this not very important.